src/http/client.rs

```diff
@@ -489,8 +490,7 @@ impl Http {
             request.body = Some(to_vec(map)?);
         } else {
             request.multipart = Some(Multipart {
-                attachment_files: Some(files.into_iter().map(Into::into).collect()),
-                upload_file: None,
+                upload: MultipartUpload::Attachments(files.into_iter().map(Into::into).collect()),
```

Comment:
The `.into_iter().map(Into::into).collect()` is actually a no-op because `files` is already a `Vec<CreateAttachment>`. I won't duplicate this note for every other instance, but in all cases they could just take `files` directly.

Reply:
Lol